cnxarchive/views/views_helpers.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
```

Comment:
Any chance we can just call this module `helpers`? Because it's already in the `views` package.